packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ Unreleased
 * Removed node_modules from .npmignore
 
 
+### Changed
+* Replaced Terra-Form with Terra-Form-Checkbox in CollapsibleMenuViewToggle
+
```

Comment:
This entry must have moved into a released section after merging master, we should move it back under Unreleased

Reply:
moved back to unreleased section : 846091737b673b34895379b71b994ce7ce6846ea